paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutInteractor.kt

```diff
@@ -108,6 +110,9 @@ internal class DefaultPaymentMethodVerticalLayoutInteractor(
         walletsState = viewModel.walletsState,
         isFlowController = viewModel is PaymentOptionsViewModel,
         onWalletSelected = viewModel::updateSelection,
+        onMandateTextUpdated = {
+            viewModel.updateMandateText(it?.resolve(viewModel.getApplication()), true)
```

Comment:
why is `showAbove` always true here? I noticed that in the screen recording, sometime the mandate text is shown below the pay button. Is that parameter just ignored?

Reply:
I _think_ everything for vertical mode is supposed to be above the button. But tabs had some other requirements.

Reply:
What I'm confused about is that the SEPA mandate shows up below the pay button even though we are setting `showAbove` to true here. Do you know why that happens? Is that the expected behavior and if so, why is there a discrepancy between this param and that behavior?

Reply:
Ahhh! That's driven by behavior for Saved Payment Methods! This change only includes changes for new payment methods!

The existing functionality for tabs + SPM just so happened to also work for vertical mode.

I'll update that and move it above the button in a follow up!

Reply:
ohhhh ok, makes sense! Thanks